mira/modeling/gromet_model.py

```diff
@@ -0,0 +1,210 @@
+from typing import List
+from itertools import count
+
+import pytz
+
+try:
+    import gromet
+except ModuleNotFoundError:
+    # For tests
+    import scripts.gromet.gromet_v1.gromet as gromet
+
+from gromet import (
+    Gromet,
+    Junction,
+    Wire,
+    UidJunction,
+    UidType,
+    UidWire,
+    Relation,
+    UidBox,
+    UidGromet,
+    ModelInterface,
+    UidMetadatum,
+    Provenance,
+    MetadatumMethod,
+    MetadatumJunction,
+    get_current_datetime,
+    Literal,
+    Val,
+    gromet_to_json,
+)
+
+from mira.modeling import Model, get_parameter_key
+
+
+__all__ = ["GroMEtModel", "model_to_gromet", "model_to_gromet_json_file"]
+
+
+class GroMEtModel:
```

Comment:
For simplicity, let's just call it `GrometModel` without further capitalization.